src/ProxyKit.Tests/ProxyKit.Tests.csproj

```diff
@@ -8,11 +8,11 @@
 
   <ItemGroup>
     <PackageReference Include="CacheCow.Client" Version="2.5.8" />
-    <PackageReference Include="MarkdownSnippets.MsBuild" Version="17.6.0" />
```

Comment:
This was causing my local build to fail.. no idea why

Reply:
What was the error message in your local build?

Reply:
C:\Users\erwin\.nuget\packages\markdownsnippets.msbuild\17.6.0\build\MarkdownSnippets.MsBuild.targets(21,6): error : No '.git' directory found. Define a `MdTargetDir`. [C:\git\ProxyKit\src\ProxyKit.Tests\ProxyKit.Tests.csproj]

Reply:
I see - I'm getting that error in ncunch because it copies things to a workspace that is not a git repo.

Reply:
https://github.com/SimonCropp/MarkdownSnippets/issues/175 

This this can stay removed and I'll use the dotnet tool verion of this instead.